.github/workflows/update.yml

```diff
@@ -0,0 +1,60 @@
+name: Data Updater
+
+on:
+  schedule:
+    - cron: '13 22 * * *'
+  workflow_dispatch:
+
+jobs:
+  update:
+    name: Update
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: true
+
+    - name: Install poetry
+      run: pipx install poetry
+
+    - uses: actions/setup-python@v5
+      with:
+        python-version: '3.9'
+        cache: 'poetry'
+
+    - name: setup python packages
+      run: poetry install
+
+    - name: Run data update script
+      run: poetry run scripts/dataupdate --verbose
+
+    - name: Push new data
+      id: auto-commit-action
+      uses: stefanzweifel/git-auto-commit-action@v5
+      with:
+        commit_message: "Automatic data update"
+
+    - name: Send GitHub Action trigger data to Slack workflow
+      id: slack
+      if: steps.auto-commit-action.outputs.changes_detected == 'true'
+      uses: slackapi/slack-github-action@v1.24.0
+      with:
+        payload: |
+          {
+            "repo_url": "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+          }
+      env:
+        SLACK_WEBHOOK_URL: ${{ secrets.HAPPY_DATABOT_SLACK_WEBHOOK }}     
```

Comment:
I think it might be preferable for this to go into the theyworkforyou channel so all the automated stuff for that is in the same place. Also, it would be nice for the output from the action was part of the message which looks like it's possible - https://stackoverflow.com/questions/59191913/how-do-i-get-the-output-of-a-specific-step-in-github-actions. That way I don't need to add a bunch of extra steps to eyeball the output to check it's done everything expected.